tutorials/deploy-xenforo-to-compute-engine/index.md

```diff
@@ -0,0 +1,267 @@
+
+    	```bash
+    	gsutil cp gs://[BUCKET_NAME]/[FILENAME] .
+	    ```
+
+    	`gsutil` is pre-installed on your Compute Engine instance. The command above copies the file you uploaded earlier to your Compute Engine instance. Replace `[BUCKET_NAME]` and `[FILENAME]` with your bucket name and the name of your XenForo installation file respectively.
+
+2. Unzip the installation file and copy the contents to Apache web root directory:
+
+	* Unzip the installation file:
+
+	    ```bash
+	    unzip [FILENAME]
+	    ```
+
+	where `[FILENAME]` is the name of your XenForo installation file.
+
+	* Copy the contents to Apache web root directory:
+
+	    ```bash
+	    sudo cp -r upload/* /var/www/html
+	    ```
+
+	* Change directory to Apache web root directory, grant file access permission XenForo requires and remove the default homepage (`index.html`):
+
+	    ```bash
+	    cd /var/www/html
+	    sudo chmod -R 777 *
+	    sudo rm index.html
+	    ```
+
+	* Open your browser and visit https://`[SERVER_EXTERNAL_IP]`. As shown in previous steps, you can find the external IP of your Compute Engine instance on the [VM Instances](https://console.cloud.google.com/compute/instances) page, in the row of the instance that you connect to. Follow the prompts on screen to configure your XenForo installation. During installation XenForo will ask you for credentials to connect to your MySQL server. Your MySQL Server and MySQL Port are localhost and 3306 respectively. Your MySQL User Name, MySQL Password and MySQL Database Name are the values you entered previously when installing MariaDB.
+
+Your XenForo website is now up and running.
+
+## Setting Up a Mailing Service
+
+XenForo by default uses a mail transfer agent (MTA), such as postfix, to send emails. The agent then utilizes an outgoing connection to common SMTP ports (25, 587, etc.) to transfer information (mails). However, many Cloud service providers, including Google Cloud Platform, block such connections for security reasons; additionally, emails sent this way are often considered spams by many email service providers, and will be dropped before getting into the inboxes of recipients. For those reasons, you should use a third-party service for sending emails from your XenForo website.
+
+[SendGrid](https://sendgrid.com/), [Mailgun](https://www.mailgun.com/), and [Mailjet](https://www.mailjet.com/) are Compute Engine’s third-party partners for sending emails. For more information on Compute Engine’s partnerships with those services, refer to [Sending Email with SendGrid](https://cloud.google.com/compute/docs/tutorials/sending-mail/using-sendgrid), [Sending Email with Mailgun](https://cloud.google.com/compute/docs/tutorials/sending-mail/using-mailgun) and [Sending Email with Mailjet](https://cloud.google.com/compute/docs/tutorials/sending-mail/using-mailjet). If you plan to use other services, follow instructions on their websites to set up SMTP Relay for your XenForo website.
```

Comment:
Change to "these services" and add third comma:

"For more information on Compute Engine’s partnerships with **these** services, refer to [Sending Email with SendGrid](https://cloud.google.com/compute/docs/tutorials/sending-mail/using-sendgrid), [Sending Email with Mailgun](https://cloud.google.com/compute/docs/tutorials/sending-mail/using-mailgun)**,** and [Sending Email with Mailjet](https://cloud.google.com/compute/docs/tutorials/sending-mail/using-mailjet)."